janitor/utils.py

```diff
@@ -150,13 +149,12 @@ def _sub_expand_grid(value, grid_index, key):  # noqa: F811
 def _sub_expand_grid(value, grid_index, key):  # noqa: F811
     """
     Expands the Index based on `grid_index`.
-
     Returns a dictionary.
     """
     name = value.name
     if not name:
         name = 0
```

Comment:
```suggestion
    name = value.name or 0
```

Reply:
Do you mind explaining the suggestion? The if stmt seems clear, I would however love to understand the suggestion. If value.name is None the or part kicks in? Is that part of python's grammar where the if clause is implied?

Reply:
It’s a trick like “x if conditions else y” but more convenient.
When “conditions” is “x” (x if x else y) equal to “x or y”.